target/linux/ipq806x/files-6.6/arch/arm/boot/dts/qcom/qcom-ipq8064-wpq864.dts

```diff
@@ -183,8 +183,11 @@
 			};
 
 			partition@1180000 {
+				compatible = "u-boot,env";
 				label = "0:APPSBLENV";
 				reg = <0x1180000 0x0080000>;
+				macaddr_ubootenv_ethaddr: ethaddr {};
+				macaddr_ubootenv_eth1addr: eth1addr {};
```

Comment:
does this actually work?

IIRC @rmilecki didn't merge such a change yet.

Reply:
Yes it works. Better than previous PR https://github.com/openwrt/openwrt/pull/15486

Reply:
Interesting. That means I can fix my other PRs to add custom names as well.

Reply:
I just looked at the code. No way this works: https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/tree/drivers/nvmem/u-boot-env.c?h=next-20240805#n119

u_boot_env_read_post_process_ethaddr is what sets the mac address. It only gets called with ethaddr.

Reply:
`ethaddr` is supported by our driver, but `eth1addr` is not supported.